### DISCUSSION TITLE
PostHog

### DISCUSSION THREAD

**Discussion by lebenitza** [keephq/keep#247] [General]
Hello,

Keep seems like a really interesting project and I want to try it out. While I understand the need for analytics to know how the project is used I feel that for the sake of transparency, a disclaimer should be made clear in Readme.md. What is PostHog, how you use it and why you need it and also offer a way to disable it.

Again, thank you for the project, we are in dire need of a really good OSS project that aggregates alerts from multiple sources properly and is easy to self host so please do not take this the wrong way. Looking forward to see how the project develops.

Cheers.

**Comment by talboren** [comment]
Hey @lebenitza! Thank you for raising this discussion 💡 

We added PostHog a while ago as we tried to understand better how users use Keep and what's interesting for them. We're going to add some general information about PostHog and how/what we use it for right away.

Just as a quick solution for that, you can use `DISABLE_POSTHOG=true` environment variable to disable everything related with PostHog (we initially added that for people who want to opt-out PostHog)

**Comment by shahargl** [comment]
@lebenitza in addition it would be awesome to hear your thoughts about Keep and how we can help more! 
You are more than welcome to join our Slack and help us build that really good OSS project that aggregates alerts :)

https://keephq.dev/slack

**Comment by talboren** [comment]
A disclaimer about PostHog was reviewed by @lebenitza and merged to main (https://github.com/keephq/keep/issues/248).

Locking this conversation for now :)
